ratpack-kafka-producer/src/main/java/smartthings/ratpack/kafka/KafkaProducerModule.java

```diff
@@ -25,6 +25,13 @@ protected void configure() {
 		Set<String> servers;
 		String clientId;
 		Long maxBlockMillis = TimeUnit.MINUTES.toMillis(1);
+		boolean enabled = true;
```

Comment:
I guess in my mind the reason for this module is to allow setting properties if so desired. I'm not that worried about the getters returning the actual driver default.  So I was thinking these values would be optional and if set we'd do the props.put and if not we wouldn't.  The getters could just return the optional value (i.e. `Optional<T>`).  This way if the driver defaults change the user will inherit them and we don't step on them.

We may also consider adding the ability to set properties directly in case people have other setting they might want to set.